standalone.go

```diff
@@ -17,9 +17,13 @@ func newStandaloneClient(opt *ClientOption, connFn connFn, retryer retryHandler)
 		return nil, err
 	}
 	s := &standalone{
-		toReplicas: opt.SendToReplicas,
-		primary:    newSingleClientWithConn(p, cmds.NewBuilder(cmds.NoSlot), !opt.DisableRetry, opt.DisableCache, retryer, false),
-		replicas:   make([]*singleClient, len(opt.Standalone.ReplicaAddress)),
+		toReplicas:     opt.SendToReplicas,
+		primary:        newSingleClientWithConn(p, cmds.NewBuilder(cmds.NoSlot), !opt.DisableRetry, opt.DisableCache, retryer, false),
+		replicas:       make([]*singleClient, len(opt.Standalone.ReplicaAddress)),
+		enableRedirect: opt.Standalone.EnableRedirect,
```

Comment:
We need to make sure that when a user sets `opt.Standalone.EnableRedirect`, it will use the `standalone` implementation.

Reply:
We also need to make sure that `EnableRedirect` and `ReplicaAddress` can't be used together.